pink/build.sh

```diff
@@ -15,8 +15,12 @@ if [[ ! -d $SLASH_PATH ]]; then
   exit 1
 fi
 cd $SLASH_PATH/slash && make
-
+UNAME_S=`uname -s`
+if test "x$UNAME_S" = "xDarwin" ; then
+  export C_INCLUDE_PATH=/opt/homebrew_include
+  export CPLUS_INCLUDE_PATH=/opt/homebrew/include
+fi
 # Compile pink
 cd $PINK_PATH
 make SLASH_PATH=$SLASH_PATH
-cd examples && make SLASH_PATH=$SLASH_PATH
+cd rondis && make SLASH_PATH=$SLASH_PATH
```

Comment:
Technically, with the addition of Slash, this file becomes redundant now?

Reply:
No need to `cd rondis && make` - rondis is built in the parent Makefile already

Reply:
ok